Detectors/ITSMFT/MFT/base/include/MFTBase/HalfDetector.h

```diff
@@ -18,7 +18,7 @@
 
 #include "TNamed.h"
 #include "TGeoVolume.h"
-#include "MFTBase/PowerSupplyUnit.h"
+//#include "MFTBase/PowerSupplyUnit.h"
```

Comment:
Please do not comment code.

Reply:
It's done.

Reply:
I see a few more. Please remove them in a subsequent PR.